content/applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_warehouse/package.rst

```diff
@@ -0,0 +1,203 @@
+  the warehouse; :guilabel:`Disposable` for packages used to ship products to customers.
+
+.. image:: package/package.png
+   :align: center
+   :alt: Display package form to create a cluster pack.
+
+Add to batch
+------------
+
+.. important::
+   The following example uses packages in a :doc:`three-step delivery
+   <../../shipping_receiving/daily_operations/delivery_three_steps>` workflow.
+
+   This workflow requires the :guilabel:`Packages`, :guilabel:`Storage Locations`, and
+   :guilabel:`Multi-Step Routes` features to be enabled in :menuselection:`Inventory app -->
+   Configuration --> Settings`.
+
+   On the warehouse configuration page (:menuselection:`Inventory app --> Configuration -->
+   Warehouses`), the :guilabel:`Outgoing Shipments` option **must** be set to :guilabel:`Pack goods,
+   send goods in output and then deliver (3 steps)`.
+
+After confirming a sales order, selecting the :guilabel:`Delivery` smart button displays the three
+tasks to complete the delivery.
+
+.. image:: package/three-step.png
+   :align: center
+   :alt: Clicking "Delivery" smart button displays the pick, pack, ship steps.
+
+To create a batch transfer, navigate to :menuselection:`Inventory app --> Operations --> Internal`
+and select the checkboxes next to the pick operations that are to be added to the batch.
+
+After the pick operations are selected, navigate to the :guilabel:`⚙️ (gear) Actions` button, and
+select :guilabel:`Add to batch` from the list of drop-down menu options.
+
+.. image:: package/create-batch.png
+   :align: center
+   :alt: Add pickings to batch.
+
+Doing so opens an :guilabel:`Add to batch` pop-up window. Click :guilabel:`Confirm` to finish
+creating the batch.
+
+.. seealso::
+   :ref:`More info about the fields in the 'Add to batch' window
+   <inventory/warehouses_storage/add-batch-transfers>`
+
+Pack items
+----------
+
+After creating the batch, open the batch transfer (:menuselection:`Inventory app --> Operations -->
+Batch Transfers`), and select a reusable package (created in the :ref:`previous step
+<inventory/warehouses_storage/package-use>`) in the :guilabel:`Destination Package` field to place
+items in the package.
+
+.. example::
+   The reusable package, `CLUSTER-PACK-1`, is assigned to the :guilabel:`Destination Package` field
+   of all items in the batch transfer, `BATCH/00003`.
+
+  .. image:: package/assign-package.png
+     :align: center
+     :alt: Assign pickings to a package through the "Destination Package" field.
+
+Alternatively, click the :guilabel:`Put in Pack` button to create a new package, and place all the
+items in the batch in that newly-created package.
+
+.. note::
+   Ensure the new package's :guilabel:`Package Use` field is set to :guilabel:`Reusable` by clicking
+   the package's :guilabel:`➡️ (right arrow)` icon to open the package form.
+
+   .. image:: package/pack-internal-link.png
+      :align: center
+      :alt: Show internal link
```

Comment:
Need a period at the end of the alt tag here, but in the next update PR: https://github.com/odoo/documentation/pull/7797

This image gets removed, so I wouldn't worry about this (;